.jenkins/jenkins_pretest.sh

```diff
@@ -8,6 +8,9 @@ set -x
 # Anaconda python
 export PATH=/usr/local/miniconda2/bin:$PATH
 
+# Clean up old .pyc files
+find ./$(git rev-parse --show-cdup) -type f -name "*.pyc" -delete
+
```

Comment:
I suggest we use Jenkins' git plugin "clean before checkout" setting, which purges untracked files:

> Clean up the workspace before every checkout by deleting all untracked files and directories, including those which are specified in .gitignore. It also resets all tracked files to their versioned state. This ensures that the workspace is in the same state as if you cloned and checked out in a brand-new empty directory, and ensures that your build is not affected by the files generated by the previous build.

I have already enabled it.